pm_uep_opt_write/test-data/Sifail.den_fmt

```diff
@@ -0,0 +1,925 @@
+ +-------------------------------------------------+
```

Comment:
This file is pretty big – it can probably be reduced to even one line and still fail the test if the code is trying to find specific header syntax.
A file that just reads "This file does not follow den_fmt syntax" is both more compact and self-documenting! Or you could use some *short* dummy data instead.

Reply:
the file is now 'this should fail'